Supervisor feedback (#80)
chhoumann/p8: report/src/system_evaluation/setup.tex

    \label{fig:experiment_room}
\end{figure}
On one end of each table is a large, rectangular window.
Furthermore, one table has a large TV screen placed near its window. 
Near the door of the room, an airconditioning controller and a large glass window is located.
Lighting is built into the ceiling, thus no hanging light will interfere with the broadcasted signals. 

> Outside the meeting room is a common area containing two high tables two high tables.

Reviewer: ```suggestion
Outside the meeting room is a common area containing two high tables.
```

Each of these tables has a small couch and two chairs placed next to it.
The meeting room and its surrounding area is depicted in Figure \ref{fig:experiment_room}.

Beacons are placed on top of the cabinet and in the three corners furthest away from the door. 
The beacons in the corners of the room are placed in a height of $2.5m$, while the center beacon on top of the cabinet is placed in a height of $1.5m$.   

> The beacons placed near the windows are located on top of a whiteboard, while the beacon in the lower left corner of the room is held in place using painters tape. 

Reviewer: ```suggestion
The beacons placed near the windows are located on top of a whiteboard, while the beacon in the lower left corner of the room is held in place using painter's tape. 
```

The phone provides auditory feedback when it has collected enough measurements for the requirements described in Section \ref{sec:first_phase} to be fulfilled.
When the information for measurement has been computed and stored on the phone, the collector is informed auditorily and continues to the next intersection in the grid. 
During the measurement the phone is kept in roughly the same height.
Measurements are collected in a setup using 4 beacons, as depicted in Figure \ref{fig:experiment_map_creation}.
The map creation process was done during out-of-office hours to ensure that the datapoint collected to create the map was collected with a minimum of background signal noise.  

\subsection{Evaluation strategy} %This is where we examine if we can classify correctly
\begin{figure}[h]
    \centering
    \includegraphics[scale=0.7]{images/experiment_setup.png}
    \caption{A sketch of the meeting room in which the experiments were carried out, decorated with beacon placements (red antenna) and measurement locations(green circles)}
    \label{fig:experiment_setup}
\end{figure}

> When performing the evaluations of the created map, we want to examine whether the system can correctly classify whether a receiving device is inside or outside the room.

Reviewer: ```suggestion
In evaluating the generated map, our aim is to determine if the system can accurately distinguish whether a receiving device is situated inside or outside the room.
```
